example/web/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "example",

```

Comment:
Maybe we should say `Animated Text Kit Example`?

Reply:
Fixed.